modules/gcp/src/main/scala/com.snowplowanalytics.snowplow.lakes/GcpApp.scala

```diff
@@ -24,12 +24,18 @@ object GcpApp extends LoaderApp[PubsubSourceConfig, PubsubSinkConfig](BuildInfo)
   override def badSink: SinkProvider = PubsubSink.resource(_)
 
   override def isDestinationSetupError: DestinationSetupErrorCheck = {
-    // Destination bucket doesn't exist
-    case e: GoogleJsonResponseException if e.getDetails.getCode === 404 =>
-      "The specified bucket does not exist"
-    // Permissions missing for Cloud Storage
+    // Bad Request
+    case e: GoogleJsonResponseException if e.getDetails.getCode === 400 =>
+      e.getDetails.getMessage
+    // Unauthorized
+    case e: GoogleJsonResponseException if e.getDetails.getCode === 401 =>
+      e.getDetails.getMessage
```

Comment:
Please can you share with me:

1. Under what circumstances do we get a 401?  What did you change in the tests to generate that error?
2. Similarly, under what circumstances do we get a 400?
3. For both cases, please give an example of `e.getDetails.getMessage`

Reply:
thanks for looking at this but I should've marked this as draft, none of the changes are needed now, I'll ping back when it's ready

Reply:
resolving as unrelated after my last change